Assignment-2/src/main/java/models/UserClass.java

```diff
@@ -0,0 +1,73 @@
+package models;
+
+import java.util.Arrays;
+
+/**
+ * User class.
+ */
+public class UserClass {
```

Comment:
Don't have to append Class with the model name.
User is fine, don't have to name UserClass

Reply:
the javadoc suggested that user is too short, initially i was using User only but later had to change .

Reply:
i have used the config of common service but still it is saying
/Users/adityapathak/Desktop/Nuclei-Assignment-Java/Assignment-2/src/main/java/models/User.java:8:       Avoid short class names like User